webapp/channels/src/i18n/en.json

```diff
@@ -5646,30 +5645,60 @@
   "user.settings.notifications.commentsInfo": "In addition to notifications for when you're mentioned, select if you would like to receive notifications on reply threads.",
   "user.settings.notifications.commentsNever": "Do not trigger notifications on messages in reply threads unless I'm mentioned",
   "user.settings.notifications.commentsRoot": "Trigger notifications on messages in threads that I start",
-  "user.settings.notifications.desktop": "Send desktop notifications",
-  "user.settings.notifications.desktop.allNoSound": "For all activity, without sound",
-  "user.settings.notifications.desktop.allSound": "For all activity, with sound",
-  "user.settings.notifications.desktop.allSoundHidden": "For all activity",
-  "user.settings.notifications.desktop.calls_sound": "Notification sound for incoming calls",
-  "user.settings.notifications.desktop.mentionsNoSound": "For mentions and direct messages, without sound",
-  "user.settings.notifications.desktop.mentionsSound": "For mentions and direct messages, with sound",
-  "user.settings.notifications.desktop.mentionsSoundHidden": "For mentions and direct messages",
-  "user.settings.notifications.desktop.sound": "Notification sound",
-  "user.settings.notifications.desktop.title": "Desktop Notifications",
-  "user.settings.notifications.email_threads": "When enabled, any reply to a thread you're following will send an email notification.",
+  "user.settings.notifications.desktopAndMobile.allDesktopButMobileMentions": "All new messages on desktop; Mentions, direct messages, and group messages on mobile",
+  "user.settings.notifications.desktopAndMobile.allDesktopButMobileNone": "All new messages on desktop; Never on mobile",
+  "user.settings.notifications.desktopAndMobile.allForDesktopAndMobile": "All new messages",
+  "user.settings.notifications.desktopAndMobile.allNewMessages": "All new messages",
+  "user.settings.notifications.desktopAndMobile.away": "Away or offline",
+  "user.settings.notifications.desktopAndMobile.differentMobileNotificationsTitle": "Use different settings for my mobile devices",
+  "user.settings.notifications.desktopAndMobile.mentionsDesktopButMobileAll": "Mentions, direct messages, and group messages on desktop; All new messages on mobile",
+  "user.settings.notifications.desktopAndMobile.mentionsForDesktopAndMobile": "Mentions, direct messages, and group messages",
+  "user.settings.notifications.desktopAndMobile.mentionsForDesktopButMobileNone": "Mentions, direct messages, and group messages on desktop; Never on mobile",
+  "user.settings.notifications.desktopAndMobile.noneDesktopButMobileAll": "Never on desktop; All new messages on mobile",
+  "user.settings.notifications.desktopAndMobile.noneDesktopButMobileMentions": "Never on desktop, Mentions, direct messages, and group messages on mobile",
```

Comment:
"user.settings.notifications.desktopAndMobile.noneDesktopButMobileMentions": "Never on desktop; Mentions, direct messages, and group messages on mobile",
  
Replace comma `,` with semicolon `;` after "Never on desktop

Reply:
done